src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -3878,6 +3885,69 @@ public override IOperation VisitTypeOf(ITypeOfOperation operation, int? captureI
             return new TypeOfExpression(operation.TypeOperand, semanticModel: null, operation.Syntax, operation.Type, operation.ConstantValue, operation.IsImplicit);
         }
 
+        public override IOperation VisitParameterInitializer(IParameterInitializerOperation operation, int? captureIdForResult)
+        {
+            var parameterRef = new ParameterReferenceExpression(operation.Parameter, semanticModel: null,
+                operation.Syntax, operation.Parameter.Type, constantValue: default, isImplicit: true);
+            VisitInitializer(rewrittenTarget: parameterRef, initializer: operation);
+            return null;
+        }
+
+        public override IOperation VisitFieldInitializer(IFieldInitializerOperation operation, int? captureIdForResult)
+        {
+            foreach (IFieldSymbol fieldSymbol in operation.InitializedFields)
+            {
+                IInstanceReferenceOperation instance = fieldSymbol.IsStatic ?
+                    null :
+                    new InstanceReferenceExpression(InstanceReferenceKind.ContainingTypeInstance, semanticModel: null,
+                        operation.Syntax, fieldSymbol.ContainingType, constantValue: default, isImplicit: true);
+                Optional<object> constantValue = fieldSymbol.HasConstantValue ? new Optional<object>(fieldSymbol.ConstantValue) : default;
```

Comment:
Why even bother calling VisitInitializer if the field has constant value? Is there a scenario I'm missing where you can have both locals (requiring a region) and a constant value here?

Reply:
I wasn't sure if `HasConstantValue = true` can be true for some error case where we can have locals. I wasn't able to come up with any such test, and if we feel this is a safe assumption, I can avoid this defensive approach.

Reply:
> Why even bother calling VisitInitializer if the field has constant value? Is there a scenario I'm missing where you can have both locals (requiring a region) and a constant value here?

We should assume anything is possible here, and it probably is in error cases. 

---
In reply to: [187417620](https://github.com/dotnet/roslyn/pull/26763#discussion_r187417620) [](ancestors = 187417620)